src/components/WalletConnect/WarnDisclaimer.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { t } from 'ttag';
+import { StyleSheet, View, Text } from 'react-native';
+import { COLORS } from '../../styles/themes';
+import { CircleInfoIcon } from '../Icons/CircleInfo.icon';
+import SimpleButton from '../SimpleButton';
+
+export const WarnDisclaimer = ({ onReadMore }) => (
```

Comment:
Why do you have the readMore as props? If the component is used only by dApp warning, it should be the same read more link.

Reply:
Refactored it, can you check again?